src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
+      const WEBAUTHN_DB_VERSION = 1;
+      const WEBAUTHN_DB_NAME = "_webauthn";
+      const WEBAUTHN_ID_TABLE = "identities";
+
+      var db = null;
+      var initPromise = null;
+
+      function initDB() {
+        return new Promise(function (resolve, reject) {
```

Comment:
And for promises, we use Q, so create the promise with that (instead of using native Promises which does not exist in IE10 or IE11)